apps/wing/src/commands/test.ts

```diff
@@ -40,7 +40,7 @@ async function testOne(entrypoint: string) {
   if (results.length === 0) {
     results.push({
       pass: true,
-      path: "(no inflight tests)",
+      path: '',
```

Comment:
was it previously misplaced? or the message "no inflight tests" will be gone if no inflight test is found?

Reply:
It's omitted now. Do you think it's important?

Reply:
I would personally expect a message telling me I have no tests. If there was no output I would assume something is broken.

Reply:
ditto, that message was helpful when I started with `wing test`

Reply:
ok!